source/Gesture Recognizer/Assets/Plugins/GestureRecognizer/Demo/Scripts/Gestures/DetectGesture.cs

```diff
@@ -54,23 +54,22 @@ private void DetectPattern()
                 trailRenderer.GetPositions(pointsToDetect);
                 strokes.Add(pointsToDetect);
             }
-            logText.text = string.Format(GestureDetectedMessage, gestureDataset.Recognize(strokes));
+            LogMessage(string.Format(GestureDetectedMessage, gestureDataset.Recognize(strokes)));
```

Comment:
This method name means is a console log, and you're using this method to change your canvas text, you can change this method name as "PrintText" or somenthing that doesn't say "Log".

Reply:
I've changed it to "ShowStatusMessage" instead of "LogMessage" and the variable name from "logText" to "statusText"